packages/EasyTest/src/Traits/MessengerAssertionsTrait.php

```diff
@@ -51,45 +53,42 @@ public static function assertCountOfMessagesSentToTransport(
     /**
      * Asserts that the given message class was dispatched by the message bus to async transport.
      *
-     * @param mixed[] $expectedProperties
-     *
-     * @phpstan-param class-string $messageClass
+     * @param class-string $messageClass
+     * @param array<string, mixed> $expectedProperties
      */
     public static function assertMessageSentToAsyncTransport(
         string $messageClass,
-        ?array $expectedProperties = null,
-        ?int $messagesCount = null
+        array $expectedProperties = [],
+        int $messagesCount = 1
     ): void {
         static::assertMessageSentToTransport($messageClass, 'async', $expectedProperties, $messagesCount);
     }
 
     /**
      * Asserts that the given message class was dispatched by the message bus to failed transport.
      *
-     * @param mixed[] $expectedProperties
-     *
-     * @phpstan-param class-string $messageClass
+     * @param class-string $messageClass
+     * @param array<string, mixed> $expectedProperties
      */
     public static function assertMessageSentToFailedTransport(
         string $messageClass,
-        ?array $expectedProperties = null,
-        ?int $messagesCount = null
+        array $expectedProperties = [],
```

Comment:
Why do we need these default values? And why doesn't it break quality tools?

Reply:
This is the final code, so default values are more useful.

This trait is not used in other packages, and PHPStan doesn't check it.

Reply:
What do you mean by final code? This method may be overrided on the app level.

Reply:
I mean that this code is not part of some abstract class or interface and will not be used for inheritance.

Using `null` values is useless.